src/include/rados/librados.h

```diff
@@ -2210,6 +2214,10 @@ CEPH_RADOS_API int rados_aio_stat(rados_ioctx_t io, const char *o,
 		                  rados_completion_t completion,
 		                  uint64_t *psize, time_t *pmtime);
 
+CEPH_RADOS_API int rados_aio_stat2(rados_ioctx_t io, const char *o,
+		                  rados_completion_t completion,
```

Comment:
IIUC, the difference between `rados_aio_stat2()` and `rados_aio_stat()` is that the former returns a higher resolution `timespec` while the latter just returns `time_t`. but the new unit tests exercising `rados_aio_stat2()` does not verify this feature? any reason why we don't want to test it?

Reply:
Add some test to compare `stat` and `stat2`. Tests for `rados_aio_stat2` need additional change, because the current `rados_aio_write_op_operate ` implemention does not set mtime correctly.
https://github.com/ceph/ceph/blob/e70ca62d9cfd5c5891d4327739ba6fc4159b5c03/src/librados/librados_c.cc#L4006